ArkhamOverlay/Data/Game.cs

```diff
@@ -176,9 +177,9 @@ public class Game : ViewModel, IGame {
                 return;
             }
 
-            if (button is ShowCardZoneButton) {
-                _logger.LogMessage($"Requesting toggle for {cardGroup.Name} {cardGroup.CardZone.Name} visibility");
-                _eventBus.PublishToggleCardZoneVisibilityRequest(cardGroup.CardZone);
+            if (button is ShowCardZoneButton showCardZoneButton) {
+                _logger.LogMessage($"Requesting toggle for {cardGroup.Name} {showCardZoneButton.CardZone} visibility");
```

Comment:
Use the card zone name here
```suggestion
                _logger.LogMessage($"Requesting toggle for {cardGroup.Name} {showCardZoneButton.CardZone.Name} visibility");
```

Reply:
oooh- nice catch!
Fixed